scripts/ci/avail-ext-doc/list-template.md

```diff
@@ -21,6 +21,6 @@ The list of extensions is also available  from the CLI. To get it, run [az exten
 az extension list-available --output table
 ```
 
-| Name | Version | Summary | Preview |
-|------|---------|---------|---------|{% for extension in extensions %}
-| [{{ extension.name }}]({{ extension.project_url }}) | {{ extension.version }} | {{ extension.desc }} | {{ extension.preview }} |{% endfor %}
+| Reference | Extension Name | Minimum Version | Description | Preview |Release Notes |
+|----|-----------|-----------------|-------------|---------|---------------|{% for extension in extensions %}
+|[{{ extension.name }}]({{ extension.project_url }}) | {{extension.name}} | {{ extension.version }} | {{ extension.desc }} | {{ extension.preview }} | [history]({{extension.history}}) |{% endfor %}
```

Comment:
I think `Minimum Version` refers to the `azext.minCliCoreVersion`. We can put extension version as the text for `Release Notes` column instead of all the same hard-coded `history`.

A split of `Reference` and `Extension Name` columns seems not necessary, they're bothing showing the extension name.

Reply:
`Minimum Version` is a bit ambigous, `Minimum CLI Version` is more clear.

Reply:
Suggest to change the `Preview` column to `Status` and the value can be `Experiment`/`Preview`/`GA`.